src/screens/FeedScreen/Header.js

```diff
@@ -289,16 +381,19 @@ const Header = ({
       height,
       headerStyle,
       showAnonymousOption,
-      onHeaderOptionClicked: () => onHeaderOptionClicked(props),
-      hideThreeDot,
       version,
       anonUserInfo: {
         colorCode: anon_user_info_color_code,
         colorName: anon_user_info_color_name,
         emojiCode: anon_user_info_emoji_code,
         emojiName: anon_user_info_emoji_name
       },
-      isPostDetail
+      isPostDetail,
+      isFollow,
+      onPressFollUnFoll,
+      onDeletePost,
+      isShowDelete,
+      isSelf
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The props passed to the `_renderProfileAnon` function have been updated to include the new props introduced in this PR. It's important to ensure that these props are used correctly within the `_renderProfileAnon` function.
```diff
+        version,
+        anonUserInfo: {
+          colorCode: anon_user_info_color_code,
+          colorName: anon_user_info_color_name,
+          emojiCode: anon_user_info_emoji_code,
+          emojiName: anon_user_info_emoji_name
+        },
+       isPostDetail,
+       isFollow,
+       onPressFollUnFoll,
+       onDeletePost,
+       isShowDelete,
+       isSelf
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->